includes/Block/Embed_Block.php

```diff
@@ -56,7 +56,7 @@ class Embed_Block extends Embed_Base {
 	 */
 	public function init() {
 		$this->register_script( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE, Tracking::SCRIPT_HANDLE ] );
-		$this->register_style( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE ] );
+		$this->register_style( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE, Embed_Base::SCRIPT_HANDLE ] );
```

Comment:
```suggestion
		$this->register_style( self::SCRIPT_HANDLE, [ self::STORY_PLAYER_HANDLE, self::SCRIPT_HANDLE ] );
```
No need to use the base class, the sub class self works just as well.

Reply:
Those are not the same.

`Embed_Block::SCRIPT_HANDLE` is `web-stories-embed-block` - for the block JS in Gutenberg

`Embed_Base::SCRIPT_HANDLE `web-stories-embed` - for the responsive CSS (and eventually JS) for embeds

Reply:
Why was `SCRIPT_HANDLE` const reused here?

Reply:
This is one of the drawbacks of `Embed_Base` doing two things...

I explained it above.

I can rename `Embed_Base::SCRIPT_HANDLE` to `Embed_Base::STYLE_HANDLE` but it's still not ideal

Reply:
Doesn't bother me that much.